frontend/src/app/app.component.ts

```diff
@@ -9,15 +9,7 @@ import { pluck } from "rxjs/operators";
   styleUrls: ["./app.component.scss"]
 })
 export class AppComponent implements OnInit {
-  messages$: Observable<Hello[]>;
-
-  constructor(private helloService: HelloServiceClient) {}
-
   ngOnInit() {
```

Comment:
You can remove unused imports and the `ngOnInit` hook since it is not used anymore.